releasenotes/notes/fix-debugger-el-isdefined-1c6b39c719c8888c.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    dynamic instrumentation: fixed an issue that caused condition expressions
+    containing ``isDefined`` to be evaluated properly.
```

```suggestion
    containing ``isDefined`` to be evaluated improperly.
```

? or could go for "incorrectly".

"fixed an issue that caused thing to happen properly" is how it reads to me, but my english sucks, so don't assume I am correct here 😅